src/Nav.js

```diff
@@ -20,6 +21,27 @@ function Nav({
   };
   const queryParams = useQuery();
   const searchQuery = queryParams.get("q");
+  useEffect(() => {
+    const unsubscribe = tinykeys(window, {
+      "$mod+KeyK": (event) => {
```

Comment:
reusability note: these two shortcuts share the same logic. they should therefore the same function. we can create a function single function and reuse it for both shortcuts.

Reply:
That makes sense. I'm going to push a commit that does that. Thanks for the feedback!